README.md

```diff
@@ -195,16 +195,16 @@ Conventions
 
 With torchaudio being a machine learning library and built on top of PyTorch,
 torchaudio is standardized around the following naming conventions. Tensors are
-assumed to have channel as the first dimension and time as the last
```

Comment:
nit: the choice of singular for channel was intentional :)

`channel` was used as "the name of the `channel` dimension" in a way similar to "the `mel` dimension", "the dimension of `freq`", `time`. Plural was avoided in order to avoid conjugating all other ones :)

It would have been a good idea to use single or double quotes instead to emphasize that this is the intended name.

As mentioned below, the number of channel would then be `n_channel`, in a way similar to `n_freq` or `n_mel`.